docs/ui/components/Search/index.ts

```diff
@@ -0,0 +1 @@
+export { Search } from './Search';
```

Comment:
Why are you exporting one file from **index.ts** instead of having the `<Search>` component in here?

In the website codebase, we put the main component inside **index.tsx**. I am not totally against the pattern you did here, mostly just curious.

Reply:
This is the pattern we use in `~ui/` directory, where component are exported one per file and `index.ts` is an aggregate, just want to stick to it. 

Also, IIRC this allows to skip one path level in the import.

Reply:
Got it. I like consistency within the codebase. Thanks for the explanation